#group name of joints that are being checked
string group_name

#list of paths to find invalid sections for
Float64Array2D[] paths
---
#invalid_sections[i] is a list of pairs; each pair is a start and end index that corresponds to an invalid section of paths[i]
#the start and end index are valid; everything between is invalid
IntArray2D[] invalid_sections
Status status
